.github/workflows/ci-linux.yml

```diff
@@ -33,157 +32,15 @@ permissions:
-        ["ubuntu-jammy"]
-      tox_packages_factors: >-
-        ["standard"]
-      docker_push_repository: ghcr.io/${{ github.repository }}/
-      logs_artifact: false
-
-  # All platforms. This duplicates the default platform, but why not,
-  # it makes it more robust regarding random timeouts.
-
-  standard:
-    if: ${{ success() || failure() }}
-    uses: ./.github/workflows/docker.yml
-    with:
-      # Build from scratch
-      free_disk_space: true
-      docker_targets: "with-system-packages configured with-targets-pre with-targets with-targets-optional"
-      # FIXME: duplicated from env.TARGETS
-      targets_pre: all-sage-local
-      targets: build doc-html
-      targets_optional: ptest
-      tox_packages_factors: >-
-          ["standard"]
-      docker_push_repository: ghcr.io/${{ github.repository }}/
-      # Make sure that all "standard" jobs can start simultaneously,
-      # so that runners are available by the time that "default" starts.
-      max_parallel: 50
-
-  standard-constraints_pkgs-norequirements:
-    if: ${{ success() || failure() }}
-    needs: [standard]
-    uses: ./.github/workflows/docker.yml
-    with:
-      # Build incrementally from previous stage (pre)
-      incremental: true
-      free_disk_space: true
-      from_docker_repository: ghcr.io/${{ github.repository }}/
-      from_docker_target: "with-targets-pre"
-      docker_targets: "with-targets-pre"
-      targets_pre: all-sage-local python3-ensure tox-ensure sagelib-tox-sagepython-constraints_pkgs-norequirements
-      tox_packages_factors: >-
-          ["standard"]
-      max_parallel: 15
-
-  standard-sitepackages:
-    if: ${{ success() || failure() }}
-    needs: [standard]
+  maximal:
     uses: ./.github/workflows/docker.yml
     with:
-      # Build incrementally from previous stage (pre)
-      incremental: true
       free_disk_space: true
-      from_docker_repository: ghcr.io/${{ github.repository }}/
-      from_docker_target: "with-targets-pre"
-      docker_targets: "with-targets with-targets-optional"
-      # FIXME: duplicated from env.TARGETS
-      targets: build doc-html
-      targets_optional: ptest
-      tox_packages_factors: >-
-        ["standard-sitepackages"]
-      # Only test systems with a usable system python (>= 3.9)
-      tox_system_factors: >-
-        ["ubuntu-jammy",
-         "ubuntu-lunar",
-         "ubuntu-mantic",
-         "debian-bookworm",
-         "debian-trixie",
-         "debian-sid",
-         "linuxmint-21.1",
-         "linuxmint-21.2",
-         "fedora-40",
-         "centos-stream-9-python3.9",
-         "almalinux-8-python3.9",
-         "gentoo-python3.10",
-         "gentoo-python3.11",
-         "archlinux-latest",
-         "opensuse-15.5-gcc_11-python3.11",
-         "opensuse-tumbleweed-python3.10",
-         "opensuse-tumbleweed"]
-      docker_push_repository: ghcr.io/${{ github.repository }}/
-      max_parallel: 8
-
-  minimal:
-    if: ${{ success() || failure() }}
```

Comment:
Why delete this? 

There are broken platforms. We may selectively turn them off until fixed.

Reply:
"gentoo-python3.10" is dead, as Gentoo doesn't ship python 3.10 any more (and 3.11 will soon be gone, too)

Reply:
> Why delete this?

See PR description.